lib/reference/selectors.mjs

```
@@ -7,21 +7,26 @@ const deprecatedHtmlTypeSelectors = new Set([
 	'applet',
 	'basefont',
 	'big',
+	'bgsound',
 	'blink',
 	'center',
 	'content',
 	'dir',
 	'font',
 	'frame',
 	'frameset',
-	'hgroup',
+	'image',
```

was silently converted to `<img>`